packages/patternfly-4/react-table/tsconfig.base.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "jsx": "react",
+    "lib": [
+      "es2017",
+      "dom"
+    ],
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "skipLibCheck": true,
+    "strict": true,
+    "strictNullChecks": false,
+    // "noImplicitAny": false,
```

Comment:
`//` is invalid in JSON, does this parse? we should probably either set it or remove it.

Reply:
removed